plugins/ssl/letsencrypt_weekly

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# monitor the number of CSRs by week for /etc/letsencrypt/csr/
+#
+# andreas perhab - andreas.perhab@wt-io-it.at
+# https://www.wt-io-it.at/
+#
+# License: GPLv2
+#
+# Parameters:
+#
+#  config
+#  autoconfig
+#
+#%# family=auto
+#%# capabilities=autoconf
+
+if [ "$1" == "autoconf" ] ; then
+    find /etc/letsencrypt/csr/ > /dev/null 2>&1 && echo "yes" || echo "no"
+elif [ "$1" == "config" ] ; then
+	echo "graph_title Letsencrypt certificates last week"
+	echo "graph_args --base 1000"
+	echo "graph_vlabel Number of certificates"
+	echo "graph_category certificates"
```

Comment:
Please use one of the *well known* category names (see http://guide.munin-monitoring.org/en/latest/reference/graph-category.html#well-known-categories). Here I would suggest `security`.

Reply:
changed category to `security` (even if that feels a bit awkward to me (the plugin does nothing for security) )